smtprelay.ini

```diff
@@ -20,6 +23,10 @@
 ;local_cert = smtpd.pem
 ;local_key  = smtpd.key
 
+; Listen on the following address for Prometheus
+; metrics exposition
+;metrics_listen = :8080
```

Comment:
I think the Prometheus metrics are a great addition, but I don't think it should be enabled by default. Can empty string mean "no Prometheus" and be the default?

Reply:
I prefer to be explicit; I would add another config option like `metrics_enabled` (set to false by default) instead of interpreting empty values

Reply:
There are already some other config strings where empty means disabled so I would prefer to keep it that way and please disabled per default since it opens another listener on all available IPs.